integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/resources/IngressResource.java

```diff
@@ -11,26 +12,22 @@
 public class IngressResource {
 
     public static Response optionsJsonEmptyEventResponse(String token, String endpoint) {
-        ByteArrayInputStream cloudEventStream = new ByteArrayInputStream("".getBytes(StandardCharsets.UTF_8));
-        return ResourceUtils.jsonRequest(token)
-                .body(cloudEventStream)
-                .contentType(ContentType.JSON)
-                .options(endpoint + "/events");
+        try (ByteArrayInputStream cloudEventStream = new ByteArrayInputStream("".getBytes(StandardCharsets.UTF_8))) {
+            return ResourceUtils.jsonRequest(token)
+                    .body(cloudEventStream)
+                    .contentType(ContentType.JSON)
+                    .options(endpoint + "/events");
```

Comment:
`.log().ifValidationFails()` is not needed here?

Reply:
`*Response` methods return a `Response` object. It is then up to the caller to handle the result and logging if failure.
Only Resource methods returning full extracted entities should log if failure.